packages/block-library/src/navigation/use-navigation-menu.js

```diff
@@ -64,6 +65,20 @@ export default function useNavigationMenu( ref ) {
 				),
 				navigationMenu,
 				navigationMenus,
+				canUserUpdateNavigationEntity: ref
+					? canUser( 'update', 'navigation', ref )
+					: undefined,
+				hasResolvedCanUserUpdateNavigationEntity: hasFinishedResolution(
+					'canUser',
+					[ 'update', 'navigation', ref ]
+				),
```

Comment:
I don't think we need to output a separate `hasResolved` thing:

```suggestion
				canUserUpdateNavigationEntity: ref
					? canUser( 'update', 'navigation', ref ) 
					&& hasFinishedResolution(
						'canUser',
						[ 'update', 'navigation', ref ]
					)
					: undefined,
```

Reply:
Nevermind, we actually need to check this in the component https://github.com/WordPress/gutenberg/pull/37286/files#diff-884dae78f1676d454c049107e9b05b2403eb042457f77f836bd170d104396b8bR350

Reply:
I'm confused. You've suggested and then marked as resolved 😕